### PR TITLE
boards: stk3600: add support (v2)

### PR DESCRIPTION
**Pull request by basilfx**
### Contribution description

This PR add supports for the Silicon Labs STK3600. The files are very similar to the SLTB001a and STK3700 already supported.

For convenience, [this is the diff](https://gist.github.com/basilfx/4bcf978fbc5e558614f28ce90001bb72) between STK3700 and STK3600 board files.

### Issues/PRs references

#8520

### CODE REVIEW

File: cpu/efm32/families/efm32lg/cpus.txt [hunk 1]
```diff
@@ -0,0 +1,72 @@
+# This file is automatically generated, and should not be changed. There is
+# propbably little reason to edit this file anyway, since it should already
```
Comment (jia200x) [inline cpu/efm32/families/efm32lg/cpus.txt]: I can deduce the typo in 'probably' is also automatically generated :)

**Comment by jia200x** [comment]
Hi @basilfx.
Maybe opening a new PR for the vendor header files and cpu related stuff? Could help with the reviews

**Comment by kYc0o** [comment]
@jia200x usually we don't review the vendor headers (or at least is what I got when we disabled all the static checks for vendor headers, maybe I'm mistaken), since they are huge and not written by the contributor of a new board. I don't know if there are cases where there are bugs there but I'd say that if so, we should patch them in subsequent PRs (which usually means to update them with a newer version).

**Comment by basilfx** [comment]
I'll fix that in another PR, since I can do that for multiple files.

ACK?

File: cpu/efm32/families/efm32lg/cpus.txt (hunk 1 above)
Comment (basilfx) [reply cpu/efm32/families/efm32lg/cpus.txt]: Addressed in #8618.

**Comment by jia200x** [comment]
>@jia200x usually we don't review the vendor headers (or at least is what I got when we disabled all the static checks for vendor headers, maybe I'm mistaken), since they are huge and not written by the contributor of a new board. I don't know if there are cases where there are bugs there but I'd say that if so, we should patch them in subsequent PRs (which usually means to update them with a newer version).

Ok! Thanks :)

**Pull request merged**